dataland-frontend/src/main.ts

```diff
@@ -32,6 +32,12 @@ function instantiateVueApp(): void {
   app.use(router);
   app.use(PrimeVue);
   app.use(pinia);
+
+  // Listen for route changes
+  router.afterEach((to) => {
```

Comment:
This is unused right now.

Reply:
In general: Shouldn't we have a different way of handling app customizations?

Reply:
Yes, this was a quick test.
In the future I would handle this utilising Vue Router's $route object to determine the current URL path and then conditionally apply classes to the menu items based on the current route